packages/server/migrations/201908021430000-channel-membership-type.js

```diff
@@ -0,0 +1,39 @@
+const {
+  CHANNEL_MEMBERSHIP_TYPE,
+} = require('../src/models/enums/channel-membership-type');
+
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.sequelize.query(`BEGIN`);
+
+      await queryInterface.addColumn('ChannelMemberships', 'type', {
```

Comment:
Instead of ``type``, how about we call this ``createdBy`` or ``createdFrom`` or similar? I prefer not using ``type`` much.

Reply:
Thanks, updated.